tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs

```diff
@@ -253,13 +253,13 @@ public static Result IsAzureServiceBusOutboundV0(this MockSpan span) => Result.F
 
         public static Result IsAzureServiceBusRequestV0(this MockSpan span) => Result.FromSpan(span)
             .Properties(s => s
-                .MatchesOneOf(Name, "ServiceBusSender.Send", "ServiceBusReceiver.Complete", "ServiceBusSessionReceiver.RenewSessionLock", "ServiceBusSessionReceiver.SetSessionState", "ServiceBusSessionReceiver.GetSessionState", "ServiceBusSender.Schedule", "ServiceBusSender.Cancel", "ServiceBusReceiver.RenewMessageLock", "ServiceBusReceiver.Abandon", "ServiceBusReceiver.Defer", "ServiceBusReceiver.DeadLetter")
+                .MatchesOneOf(Name, "servicebus.publish", "servicebus.settle", "client.request")
                 .Matches(Type, "http"))
             .Tags(s => s
                 .Matches("az.namespace", "Microsoft.ServiceBus")
                 .IsPresent("az.schema_url")
                 .IsOptional("messaging.destination.name")
-                .IfPresentMatchesOneOf("messaging.operation", "publish", "receive", "settle")
+                .IfPresentMatchesOneOf("messaging.operation", "publish", "settle")
```

Comment:
I didn't see `receive` in any of the snapshots for `messaging.operation` before/after this so I removed it - unsure if that is the right thing to do, but thought it may be confusing to see `client.request` in there and no name with `receive` in it.

Reply:
Good find, I think it was accidentally included there on my first implementation. Thanks for cleaning it up!